src/coreclr/tools/Common/Internal/Text/Utf8StringBuilder.cs

```diff
@@ -16,8 +16,7 @@ public Utf8StringBuilder()
         {
         }
 
-        // TODO: This should return ReadOnlySpan<byte> instead once available
-        public byte[] UnderlyingArray => _buffer;
+        public ReadOnlySpan<byte> UnderlyingArray => _buffer;
```

Comment:
```suggestion
```
Is this actually used anywhere? It may be better to delete this method instead.

Reply:
This is what I ask myself everytime I see something in a Common folder 😄 

It seems like it's not, thanks!

Reply:
update: It was used by `DwarfEhFrame`..
![image](https://github.com/dotnet/runtime/assets/26825431/5d2da28b-e03d-4fa9-a0f9-693d9dc9b95e)

I looked at the local and made assumption based on the suffix.. oops.